lib/app/service/vendor.go

```diff
@@ -769,8 +794,8 @@ func resourcesFromPath(root string, includePatterns []string, ignorePatterns []s
 		}
 		resourceFile, err := resources.NewResourceFile(path)
 		if err != nil {
-			return trace.Wrap(err, "failed to parse resource file %q: %v",
-				utils.TrimPathPrefix(path, root, defaults.ResourcesDir), err)
+			return trace.Wrap(err).AddUserMessage("%v", err).
```

Comment:
Are you worried that this loses the error context so it is additionally logged as a string?

Reply:
No, the error's call stack is logged in full, but the users get a nice message (like, the example on the trace PR).